opm/simulators/flow/FlowMainEbos.hpp

```diff
@@ -545,35 +544,6 @@ namespace Opm
         const Schedule& schedule() const
         { return ebosSimulator_->vanguard().schedule(); }
 
-
-        // Run diagnostics.
-        // Writes to:
-        //   OpmLog singleton.
-        void runDiagnostics()
-        {
-            if (!this->output_cout_) {
-                return;
-            }
-
-            // Run relperm diagnostics if we have more than one phase.
-            if (FluidSystem::numActivePhases() > 1) {
-                RelpermDiagnostics diagnostic;
-                if (mpi_size_ > 1) {
-#if HAVE_MPI
-                    this->grid().switchToGlobalView();
-                    static_cast<ParallelEclipseState&>(this->eclState()).switchToGlobalProps();
-#endif
-                }
-                diagnostic.diagnosis(eclState(), this->grid());
-                if (mpi_size_ > 1) {
```

Comment:
Why was the global grid passed to the diagnostics here? I have tested this PR both in serial and in parallel and it seems to work perfectly without a similar logic.

Reply:
the diagnostics needs the entire grid to do the checks. if you only pass the local part, you get *different* diagnostics.

Reply:
>  you get different diagnostics.

Can you point me where the difference occur? I though it just checked the tables and the endpoints.

Reply:
The details alludes me, it was a year ago or so. But you got different output on parallel and serial norne. In particular you got some warnings in parallel that did not spew in serial.

Reply:
ok, I will checkout norne, thanks

Reply:
Note that this could also be related to the mapper. Maybe we didn't have the local mapper available or something. I just remember I did this change to yield the same results in parallel and serial (https://github.com/opm/opm-simulators/pull/2309 - not a lot of additional info there..)

Reply:
Norne does not give any warnings from the relperm diagnostics neither in serial or in parallel. Maybe @bska has fixed the underlying issue  during his re-factoring?

Reply:
that is certainly possible. if you get the same results and are confident then just go ahead. i know it fixed an issue back then, but if it's the same now..

Reply:
The relperm diagnostics for Model 2 gives the same output for this PR and master both in serial and in parallel. 

I however notices that the simulator gives to many warnings like 

> Warning: Equilibration region 1 has no active cells

in parallel. Since it gives a warning if a partition has no active cells. I will provide a fix in another PR.

Reply:
> The relperm diagnostics for Model 2 gives the same output for this PR and master both in serial and in parallel.

Very good.  In that case I'm satisfied that this PR is good to go.  I'll merge into master.